### PR TITLE
Add support for CSV formatter

### PR DESCRIPTION
**Pull request by hauleth**
Issue #22

### CODE REVIEW

**Comment by hauleth** [comment]
It would worth consideration to merge `Formatters` into single modules as it would simplify adding new ones and testing them.

**Comment by simonprev** [comment]
Wow this is an awesome PR! I will open an issue to refactor all formatter tests to use the same `for` "loop" to generate tests with relevant name. 👍 I made a few comments, but this will be good to merge after 👍 Thank you @hauleth

File: lib/web/plugs/movement_context_parser.ex [hunk 1]
```diff
@@ -1,17 +1,10 @@
 defmodule Accent.Plugs.MovementContextParser do
   use Plug.Builder
 
+  import Langue
```
[the remark this answers is not in the document]
Comment (hauleth) [reply lib/web/plugs/movement_context_parser.ex]: I have used `import` to reduce amount of changes introduced in the PR (as I just moved private methods to separate module).

**Comment by simonprev** [comment]
I will merge this but future improvement on the formatter could add "value type" parsing to add hint on the content of the string: float, integer, null value, empty value, etc

👍

**Pull request merged**